scenarios/aws/microVMs/microvms/provision.go

```diff
@@ -27,9 +28,38 @@ func GetSudoPassword(ctx *pulumi.Context, isLocal bool) pulumi.StringOutput {
 	return SudoPasswordRemote
 }
 
+func sharedFSMountpoint() string {
+	return filepath.Join(GetWorkingDirectory(), "kmt-ramfs")
+}
+
+const createRAMFS = `mkdir -p %[1]s && \
+sudo mount -t tmpfs -o size=%[2]s tmpfs %[1]s \
+`
+
+const exportRAMFS = `sudo sh -c 'echo "%[1]s 100.0.0.0/8(ro,no_root_squash,no_subtree_check)" >> /etc/exports && exportfs -a '`
```

Comment:
If these are only used in the one function, let's put them there. It makes them easier to find.